addons/mail/static/src/widgets/form_renderer/form_renderer.js

```diff
@@ -31,6 +31,13 @@ FormRenderer.include({
         // Do not load chatter in form view dialogs
         this._isFromFormViewDialog = params.isFromFormViewDialog;
     },
+    /**
+     * @override
+     */
+    async willStart() {
+        await this._super(...arguments);
+        await owl.Component.env.messagingCreatedPromise;
```

Comment:
Delaying the render of the form view because of messaging is way more problematic for user experience than delaying discuss for example.

If there is an alternative solution here it should be preferred.

Fortunately I see no occurrence of "messaging" in this file so it should be fine to not do it here?